xl-v1-multipart.go

```diff
@@ -209,7 +210,6 @@ func (xl xlObjects) putObjectPartCommon(bucket string, object string, uploadID s
 		return "", toObjectErr(err, minioMetaBucket, partPath)
 	}
 
```

Comment:
I thought that the comment doesn't add much after I moved `readXLMetadata` right after writing the part to the disks. Do you think we should add it back?